### PR TITLE
Des 732

### PR DESCRIPTION
**Pull request by mhomolak**
Placeholder PhoneNumber component

### CODE REVIEW

File: demo/demoPages/PhoneNumberPage.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+const PhoneNumberPage = () => (
+  <div className="displaySection">
+    <h1>Phone Number</h1> <br/>
+    <div elementContainer>
+      <div className="code">
+        <h3>Props</h3>
+        <h4>Optional:</h4>
+          <ul>
+            <li className="li-props">labelText:String === {`<PhoneNumber labelText="Your label" />`}</li>
```
Comment (StommePoes) [inline demo/demoPages/PhoneNumberPage.js]: should this be made required? Labels in *general* are for our inputs (or should be).

File: src/PhoneNumber/rrui.css [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+/* Not introducing an animation on `.rrui__select__option`
+   because it would introduce a little clutter due to
+   focused option animation in autocomplete mode */
+.rrui__select__selected
+{
+	transition         : background-color 30ms ease-out, color 30ms ease-out;
+	-webkit-transition : background-color 30ms ease-out, color 30ms ease-out;
+}
+
+.rrui__select__selected-content
```
Comment (StommePoes) [inline src/PhoneNumber/rrui.css]: will check in IE

File: demo/demoPages/PhoneNumberPage.js (hunk 1 above)
Comment (mhomolak) [inline demo/demoPages/PhoneNumberPage.js]: This is only going to be used as a "test" component for PI in staging.  For our actual implementation I agree, it should be required but for this I didn't set it as such.

File: src/PhoneNumber/rrui.css (hunk 2 above)
Comment (StommePoes) [inline src/PhoneNumber/rrui.css]: Looked good in IE even with the flex.

**Pull request merged**